authorizations/authorization-common/src/main/java/org/apache/gravitino/authorization/jdbc/JdbcAuthorizationProperties.java

```diff
@@ -34,7 +34,7 @@ public JdbcAuthorizationProperties(Map<String, String> properties) {
   }
 
   private void check(String key, String errorMsg) {
-    if (!properties.containsKey(key) && properties.get(key) != null) {
+    if (!properties.containsKey(key)) {
```

Comment:
This is not correct what happens if the property is null?

Reply:
in case like this is best you make a little chart 2x2 with the key existing and not existing vs the property being null or not, you can work out the correct logic for from that.